File: VBNetCore.Mvvm/Controls/PopupControl.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows;
using System.Windows.Controls;

namespace NetCore.Mvvm.Controls
{
    public class PopupControl : ContentControl
    {
        static PopupControl()
        {
            DefaultStyleKeyProperty.OverrideMetadata(typeof(PopupControl), new FrameworkPropertyMetadata(typeof(PopupControl)));
        }
    }
}

File: VBNetCore.Mvvm/Controls/PopupControl.vb
Imports System.Windows
Imports System.Windows.Controls

Namespace VBNetCore.Mvvm.Controls
    Public Class PopupControl
        Inherits ContentControl
        Shared Sub New()
            Call DefaultStyleKeyProperty.OverrideMetadata(GetType(PopupControl), New FrameworkPropertyMetadata(GetType(PopupControl)))
        End Sub
    End Class
End Namespace
